guides/common/modules/proc_creating-an-organization.adoc

```diff
@@ -11,6 +11,15 @@ To use the CLI instead of the {ProjectWebUI}, see xref:cli-creating-an-organizat
 . In the *Label* field, enter a unique identifier for the organization.
 This is used for creating and mapping certain assets, such as directories for content storage.
 Use letters, numbers, underscores, and dashes, but no spaces.
+ifdef::satellite[]
+. Optional: If you do not wish to enable Simple Content Access (SCA), uncheck the *Simple Content Access* checkbox.
+For more information on SCA, see https://access.redhat.com/articles/simple-content-access[Simple Content Access].
++
+[NOTE]
+====
+We do not recommend disabling SCA as the entitlement mode is deprecated.
```

Comment:
```suggestion
{Team} does not recommend disabling SCA as the entitlement mode is deprecated.
```

Reply:
I think "entitlement mode" is more correct than "the entitlement mode."  I haven't heard of anyone saying it with "the."